components/support/nimbus-fml/src/frontend.rs

```diff
@@ -120,6 +120,7 @@ pub(crate) struct ImportBlock {
 
 #[derive(Debug, Deserialize, Serialize, Clone)]
 #[serde(deny_unknown_fields)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
I'd probably put this further up: i.e. above `ManifestFrontend`.

Reply:
It didn't change the case when it was up on `ManifestFrontend`, so it looks like it needs to stay here